R/read.LPJ_input.R

```diff
@@ -1,33 +1,35 @@
 #' read.LPJ_input
-#' 
+#'
 #' Reads a LPJmL input file and converts it to a LPJ-object
-#' 
+#'
 #' This function reads in LPJ-input files, using its header information. So
 #' far, tested for landuse input.
-#' 
+#'
 #' @usage read.LPJ_input(file_name, out_years=c("y1995","y2005"),
-#' namesum=FALSE, four_d=FALSE,swap="little")
+#' namesum=FALSE, four_d=FALSE, ncells=59199, swap="little")
```

Comment:
please delete the @usage row completely (not needed anymore)

Reply:
Done!

Reply:
done!